configure.ac

```diff
@@ -106,15 +106,15 @@ AC_SUBST(XMLSEC_LIBS)
 # found in the LASSO_CFLAGS. Save and restore CFLAGS and CPPFLAGS.
 saved_CFLAGS=$CFLAGS
 saved_CPPFLAGS=$CPPFLAGS
-CFLAGS="$CFLAGS $pkg_cv_LASSO_CFLAGS"
-CPPFLAGS="$CPPFLAGS $pkg_cv_LASSO_CFLAGS"
+CFLAGS="$CFLAGS $pkg_cv_LASSO_CFLAGS $GLIB_CFLAGS $XMLSEC_CFLAGS"
```

Comment:
wouldn't it be more consistent to use `$pkg_cv_GLIB_CFLAGS`? It's probably the same values, it would just look more similar..

Reply:
@jhrozek thanks for following up. I have to confess I know almost nothing about this build system. I'll go ahead and replace all my instances of `$GLIB_CFLAGS` with `$pkg_cv_GLIB_CFLAGS` then? Same for `$XMLSEC_CFLAGS` -> `$pkg_cv_XMLSEC_CFLAGS`?

Reply:
you know I was probably being over pedantic. Let's get the patch merged, I don't see anything wrong about it tbh. If others do, they had ~20 days to review the PR and can always send another PR on top :-P